sdcm/utils/nemesis.py

```diff
@@ -123,10 +129,14 @@ def create_job_files_from_template(self) -> list[Path]:
             job_file_name = f"{self.base_job}-{cls}-{self.backend}.jenkinsfile"
             nemesis_job_groovy_source = Template(self.nemesis_job_template).render(
                 {
-                    "nemesis_longevity_config": config_name,
-                    "backend": self.backend,
-                    "region": self.BACKEND_TO_REGION.get(self.backend, "eu-west-1"),
-                    "additional_params": additional_params,
+                    "params": {
+                        "backend": self.backend,
+                        "region": self.BACKEND_TO_REGION.get(self.backend, "eu-west-1"),
```

Comment:
Looks unsafe, default region must be tied to the backend

Reply:
It is, this is fallback when there is no region tied to a backend. I can remove the default if you want.